Package.swift

```diff
@@ -1,10 +1,14 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.5
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "AppSyncRealTimeClient",
+    platforms: [
+        .iOS(.v13),
```

Comment:
wait i think... iOS AppSync SDK still uses this, will that impact the SDK to take on new versions of this package?

Reply:
https://github.com/awslabs/aws-mobile-appsync-sdk-ios

Reply:
Yes, this dependency has to match the amplify-ios platform requirements to be compatible.